server/testutils/containere2e/containere2e.go

```diff
@@ -35,12 +56,22 @@ func NewE2ETestPlugin(t *testing.T) (*mmcontainer.MattermostContainer, *sqlstore
 		"tenantid":                   "tenant-id",
 		"webhooksecret":              "webhook-secret",
 	}
+
 	mattermost, err := mmcontainer.RunContainer(ctx,
 		mmcontainer.WithPlugin(filename, "com.mattermost.msteams-sync", pluginConfig),
 		mmcontainer.WithEnv("MM_MSTEAMSSYNC_MOCK_CLIENT", "true"),
+		mmcontainer.WithTestingLogConsumer(t),
 	)
 	require.NoError(t, err)
 
+	// TODO: This won't be required after jespino gets https://github.com/testcontainers/testcontainers-go/pull/2073 merged.
```

Comment:
I saw your PR, @jespino! This should help us now before we can more easily configure as you have above.